app/forms/questionnaire_form.py

```diff
@@ -440,15 +445,39 @@ def _option_value_in_data(
 
 def get_answer_fields(
     question: QuestionSchemaType,
-    data: Union[None, MultiDict[str, Any], Mapping[str, Any]],
+    data: MultiDict[str, Any] | Mapping[str, Any] | None,
     schema: QuestionnaireSchema,
     answer_store: AnswerStore,
     list_store: ListStore,
-    metadata: Optional[MetadataProxy],
+    metadata: MetadataProxy | None,
     response_metadata: Mapping[str, Any],
-    location: Union[Location, RelationshipLocation, None],
+    location: Location | RelationshipLocation | None,
+    progress_store: ProgressStore | None,
 ) -> dict[str, FieldHandler]:
     list_item_id = location.list_item_id if location else None
+
+    routing_path_block_ids: Optional[dict[tuple, list[str]]] = {}
```

Comment:
This isn't optional, is always a dict, is also initialised as empty dict.

Reply:
Updated 👍